src/zope/testbrowser/tests/test_browser.py

```diff
@@ -1079,9 +1121,15 @@ def test_multiple_classes(self):
 
 
 def test_suite():
-    return doctest.DocTestSuite(
-        checker=zope.testbrowser.tests.helper.checker,
-        optionflags=doctest.NORMALIZE_WHITESPACE | doctest.ELLIPSIS)
+    suite = unittest.TestSuite()
+    suite.addTests([
+        unittest.makeSuite(TestDisplayValue),
+        unittest.makeSuite(TestMechRepr),
```

Comment:
Instead of repeating `unittest.makeSuite()` for every test class, I recommend `unittest.defaultTestLoader.loadTestsFromName(__name__)`, which finds all test classes automatically.